src/channels.js

```diff
@@ -112,6 +113,21 @@ export const readChannels = async (limit = 1000) => {
 	return supabase.from('channels').select('*').limit(limit).order('created_at', {ascending: true})
 }
 
+/**
+ * Returns a paginated list of channels.
+ * @returns {Promise<ReturnObj>}
+ */
+
+export const browseChannels = async ({page, limit}) => {
```

Comment:
Now we have `readChannels(limit)` and `browseChannels({page, limit}`.

How putting them together in `readChannels({limit, page})` or `readChannels(limit, page)`?

Reply:
yes sounds cool too, idk

Reply:
for example what if we browse "tracks" in a radio channel, and want to order them by title?
- do we want that?
- do we prefer having "just a search"?

Reply:
Finally, these params (`page`, `limit`) will all be also bound to the URL, I guess this won't be in the sdk though

Reply:
OK, let's change `readChannels` then.

There must be many smart abstractions we can do. I feel for now we can experiment in r4-app by writing `sdk.supabase.from()` queries directly? And once we know the patterns move them to the sdk.